config/routes.rb

```diff
@@ -44,6 +44,7 @@
     resources :rollbacks, only: %i(create)
     resources :tasks, only: %i(show) do
       collection do
+        get '' => 'tasks#index', as: :index
```

Comment:
I think you can change the `only` option to include `index` and get the same result.